### DISCUSSION TITLE
The mjai.ekyu.moe site is down, Argo Tunnel error

### DISCUSSION THREAD

**Discussion by chucklu** [Equim-chan/mjai-reviewer#140] [General]
When try to access this site, got the error from Cloudflare, Argo Tunnel error
![image](https://github.com/Equim-chan/mjai-reviewer/assets/3004057/927fb8d1-8036-4d44-aaf5-395ac6e6bd6e)

**Comment by Equim-chan** [comment]
The website is down due to an emergent power failure.

Recovery time is currently unknown.
